### PR TITLE
added: readonly access bundle

### PR DESCRIPTION
**Pull request by fungus75**
Hi Kevin, would be great if you add my bundle/plugin.
I think it's quite nice, but it is still alpha...

cu
 Rene

### CODE REVIEW

File: _store/derpilzcc-readonly-access-bundle.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+cd /kimai/var/plugins/
+git clone https://github.com/fungus75/ReadOnlyAccessBundle.git
+```
+
+
+You could also [download it as zip](https://github.com/fungus75/ReadOnlyAccessBundle/archive/master.zip) and upload the directory via FTP:
+
+```
+/kimai/var/plugins/
+├── CustomCSSBundle
```
Comment (kevinpapst) [inline _store/derpilzcc-readonly-access-bundle.md]: Copy&Paste

File: _store/derpilzcc-readonly-access-bundle.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+price: "0"
+download: https://github.com/fungus75/ReadOnlyAccessBundle/archive/master.zip
+github: https://github.com/fungus75/ReadOnlyAccessBundle
+screenshot: 
+  - https://raw.githubusercontent.com/fungus75/ReadOnlyAccessBundle/master/screenshot.jpg
+tags:
+  - plugin
+---
+
+This bundle adds a special admin screen where the role can be assigned to an existing Kimai user.
```
Comment (kevinpapst) [inline _store/derpilzcc-readonly-access-bundle.md]: Can you explain the plugin a bit more, I don't understand what it does from that sentence.

**Comment by fungus75** [comment]
Hi Kevin, hopefully now it is clear?

**Comment by kevinpapst** [comment]
Much better 👍 
Can you have a look at my comments in your repo: 
https://github.com/fungus75/ReadOnlyAccessBundle/issues/1

**Comment by fungus75** [comment]
Have adjusted the "installation"-Part a little bit, because now no own permission and no local.yuml is neccessary.

**Comment by kevinpapst** [comment]
As posted on your repository, could you choose another icon an use the same one here as well?!
Right now both are copy&paste from another bundle and sonÄt match between store and Kimai.
Bildschirmfoto 2019-08-12 um 23 16 04

Bildschirmfoto 2019-08-12 um 23 18 14

**Comment by kevinpapst** [comment]
Thanks Rene for your contribution to Kimai 🎉

**Comment by kevinpapst** [comment]
https://www.kimai.org/store/derpilzcc-readonly-access-bundle.html

**Pull request merged**